docs/source/developing/writing-docs.adoc

```diff
@@ -101,4 +172,7 @@ $ cd ..
 $ rake build
 ----
 
-If the build completes successfully, the site is available under _preview/openshift-origin/latest/welcome/index.html_.
+If the build completes successfully, the site is available under *_preview/openshift-origin/latest/welcome/index.html_*.
+
+OpenShift runs a nightly build in which they update the documentation. After you check-in the updated tarball,
```

Comment:
So the workflow is that the tarball gets created on request and then gets consumed with 24 h by doc.openshift.org